lrtable/src/lib/statetable.rs

```diff
@@ -617,7 +616,9 @@ mod test {
           ").unwrap();
         let sg = pager_stategraph(&grm);
         let st = StateTable::new(&grm, &sg).unwrap();
-        assert_eq!(st.actions.len(), 8);
+
+        let length = usize::from(grm.tokens_len()) * usize::from(sg.all_states_len());
```

Comment:
Conventionally this is `len` (or `l`).

Reply:
Ok, guess I was still in Python world, where `len` is reserved.